challenges/prototypes.js

```diff
@@ -6,28 +6,47 @@
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+function CuboidMaker(CuboidAttribute){
+  this.length = CuboidAttribute.length;
+  this.width = CuboidAttribute.width;
+  this.height = CuboidAttribute.height;
+}
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
-  
   Formula for cuboid volume: length * width * height
 */
+  CuboidMaker.prototype.volume = function(length,width,height){
```

Comment:
in your volume method, you have passed 3 params to it.  Are you using it anywhere?